userfrosting/locale/fr_FR.php

```diff
@@ -76,7 +83,6 @@
 	"ACCOUNT_PASSWORD_UPDATED" => "Le mot de passe du compte a été mis à jour",
 	"ACCOUNT_EMAIL_UPDATED" => "L'adresse e-mail du compte a été mise à jour",
 	"ACCOUNT_TOKEN_NOT_FOUND" => "Ce jeton n'existe pas / Le compte est déjà activé",
-	"ACCOUNT_USER_INVALID_CHARACTERS" => "Le nom d'utilisateur doit contenir seulement des caractères alphanumériques",
 	"ACCOUNT_DELETE_MASTER" => "Vous ne pouvez pas supprimer le compte principal !",
 	"ACCOUNT_DISABLE_MASTER" => "Vous ne pouvez pas désativer le compte principal !",
```

Comment:
I believe there's a typo here. I know it's not part of what you added, but could you commit a fix so we can merge it in at the same time?